controllers/storagecluster/odfinfoconfig.go

```diff
@@ -137,6 +137,12 @@ func getOdfInfoData(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCl
 		return "", err
 	}
 
+	annotation := map[string]string{}
+	ocsApiEndpointExportedAddress := storageCluster.GetAnnotations()[ocsv1a1.OcsApiEndpointExportedAddress]
```

Comment:
What is the maximum length of string supported by annotation? Would it be enough to capture the address?

Reply:
Yes, I don't think there is a restriction on the value of the annotation key. We capture the onboarding ticket on storageConsumer CR that is 800+ characters

Reply:
I think length restriction only apply to labels, not annotations